lib/dialects/abstract/index.js

```diff
@@ -42,7 +42,14 @@ AbstractDialect.prototype.supports = {
   migrations: true,
   upserts: true,
   constraints: {
-    restrict: true
+    restrict: true,
+    addConstraint: true,
```

Comment:
why `addConstraint` and `dropConstraint` are part of this, is it for the fact some dialect doesn't support adding / dropping constraints